### DISCUSSION TITLE
Send User Invite from App

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8629] [Help]
Hey,

is it possible to add users with a name & a email from inside your app?

Use Case is that we have contracts which need approval from certain people that we don't know in the first place.
The User should be able to enter a name and a email which sends out the app invitation.

**Comment by melohagan** [comment]
Hey @kasgithub8 

Have you tried using the public API? https://docs.budibase.com/reference/post_users

You could maybe combine that with the SMTP email step in an automation to notify them that they've been added.

**Reply by kasgithub8** [reply]
> Hey @kasgithub8
> 
> Have you tried using the public API? https://docs.budibase.com/reference/post_users
> 
> You could maybe combine that with the SMTP email step in an automation to notify them that they've been added.

Okay @melohagan, so I actually tried it out earlier in the suggested "Insomnia" program but it didn't work.
Using the same settings on the page you gave me it worked.
Im gonna see what I can do and if that's possible for me. Thank you, for now! 😄
